routes/middlewares/festival/lol/post/postLolVote.ts

```diff
@@ -0,0 +1,51 @@
+import CustomError from '@Middleware/error/customError';
+import { NextFunction, Request, Response } from 'express';
+
+import LolTeam from '@Model/lolTeam.model';
+import LolVote from '@Model/lolVote.model';
+
+const postLolVote = async (req: Request, res: Response, next: NextFunction) => {
+  const team_pk: number = req.body.team_pk;
```

Comment:
```suggestion
  const { team_pk }: Body = req.body;
```
새로운 방법 시도, Body 라는 타입을 따로 만들어서 관리해주면 좋을듯.